crypto/ct/ct_sct.c

```diff
@@ -317,12 +317,11 @@ int SCT_set_source(SCT *sct, sct_source_t source)
 
 int SCT_LIST_set_source(const STACK_OF(SCT) *scts, sct_source_t source)
 {
-    int i, ret = 1;
+    int i, ret = 0;
 
     for (i = 0; i < sk_SCT_num(scts); ++i) {
-        ret = SCT_set_source(sk_SCT_value(scts, i), source);
-        if (ret != 1)
-            break;
+        if (SCT_set_source(sk_SCT_value(scts, i), source))
```

Comment:
Ah so I do use this in another branch (https://github.com/RJPercival/openssl/commit/57a1a07e2b75e95534f5dcdc403d95fa9b809538), which I haven't created a PR for (because I'm not happy with how it's implemented). 

The expected use is to set the `log` field on every SCT in the list from a single log store. This isn't necessary if you're validating the SCTs, because the `log` field gets set during validation. However, there are cases when SCTs aren't validated (e.g. when using "openssl x509 -text" to print a certificate"), but it would still be nice to know which log each one comes from. IN the example above, knowing the log it's from allows you to print the log name, rather than just the log ID (which is far less recognisable).